spanner/tests/system/test_system.py

```diff
@@ -960,6 +960,12 @@ def test_read_w_ranges(self):
         snapshot = session.snapshot(read_timestamp=committed, multi_use=True)
         all_data_rows = list(self._row_data(ROW_COUNT))
 
+        single_key = KeyRange(start_closed=[START], end_open=[START+1])
```

Comment:
`START+1` -> `START + 1` here and below (sorry for being a human linter)

Reply:
I'll change it like so to comply with PEP8.  Making the spaces inside slices makes it ugly.